addons/mail/tests/test_link_preview.py

```diff
@@ -21,9 +22,40 @@ def _patched_get_html(*args, **kwargs):
     </head>
     </html>
     """
+    # To handle chunks read on stream requests
+    response.raw = io.BytesIO(response._content)
     response.headers["Content-Type"] = 'text/html'
     return response
 
+def _patched_get_html_without_og_title(*args, **kwargs):
+    response = requests.Response()
```

Comment:
Seems there is duplicated code between the various mocks ... wouldn't it be possible to have a single method generating the response, taking parameters ?